lib/phoenix/view.ex

```diff
@@ -10,18 +10,26 @@ defmodule Phoenix.View do
 
   ## Examples
 
```

Comment:
@Gazler I also changed `lib/web`* etc, to `lib/your_app_web`* etc. It seems to make more sense since we use snippets of supposedly `YourAppWeb`?

Lmk